MCDataProducts/inc/CosmicLivetime.hh

```diff
@@ -42,6 +43,15 @@ namespace mu2e {
         _livetime = _primaries / (M_PI * _area * _fluxConstant * (EfToOneMinusGamma - EiToOneMinusGamma) / (1. + eslope));
     }
 
+    CosmicLivetime( float livetime ):
+      _primaries(0),
+      _area(0),
+      _lowE(0),
+      _highE(0),
+      _fluxConstant(0) {
+      _livetime = livetime;
+    }
+
```

Comment:
Here can't you initialize `_livetime` as you do for the other parameters? However, I am not sure that in CRY you don't have those numbers. It must be using a formula internally to calculate the livetime no? Probably primaries could be misleading since you don't generate protons in CRY, but it must assume an area, an energy interval and a flux constant to calculate the livetime no?

Reply:
I don't see a big point in storing these variables.
We won't be able to reconstruct the livetime based on these quantities. 
However, I can add them for completeness.

Reply:
Ok, your call. I personally think it would be nice to store them somewhere for reference, but if you think it's misleading just keep 0 (or -1)

Reply:
My personal take is that any information which is available and correct should be loaded, even if we don't know of a use case today.  If accessing that info is a lot of work, then that work can be deferred to a future PR, but it should not be lost (ie add FIXME and an issue).